src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
-import React from 'react';
+import { Comment } from '../../types/comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {
+      comments.map(
+        (comment) => <CommentInfo key={comment.id} comment={comment} />,
+      )
+    }
```

Comment:
I suggest the next formatting
to have HTML on its own line
```suggestion
    {comments.map(comment => (
      <CommentInfo key={comment.id} comment={comment} />
    ))}
```

Reply:
Yes, looks better